.github/workflows/test.yml

```
@@ -0,0 +1,28 @@
+name: test
+
+on:
+  - push
+  - pull_request
```

With this configuration, the pipelines are executed twice for branches in this repo that are used to create a PR (as it is the case for this particular PR / branch `actions`).

Based on https://docs.github.com/en/actions/reference/events-that-trigger-workflows, I suggest:

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

---

Okay, that's fair. Is my understanding correct that workflows are branch-specific, i.e. we'd have to cherry-pick this commit to the maint-1.x branch and replace the master branch reference with maint-1.x?

---

Yes, this is correct. We need to apply a similar commit to the `maint-1.x` branch where we replace `master` with `maint-1.x`.

---

Thanks for the confirmation. Using your suggestion and pushing an additional commit to the maint-1.x branch.